demos/2d/space_shooter/engine.cfg

```diff
@@ -10,6 +10,10 @@ game_state="res://game_state.gd"
 
 [display]
 
+driver="GLES2"
+fullscreen=false
+text_edit_idle_detect_sec=3
+stretch_aspect="ignore"
```

Comment:
Hm, it's weird that they got added, normally the config only shows custom values.